spec/forms/teacher_interface/delete_upload_form_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "rails_helper"
+
+RSpec.describe TeacherInterface::DeleteUploadForm, type: :model do
+  describe "validations" do
+    subject(:form) { described_class.new(confirm:, upload:) }
+
+    context "when the fields are blank" do
+      let(:confirm) { "" }
+      let!(:upload) { "" }
```

Comment:
```suggestion
      let(:upload) { "" }
```

These can be lazy as they don't create any objects.